packages/browser-sdk/src/feature/features.ts

```diff
@@ -138,14 +145,38 @@ type context = {
 
 export const FEATURES_EXPIRE_MS = 30 * 24 * 60 * 60 * 1000; // expire entirely after 30 days
 
-const localStorageCacheKey = `__bucket_features`;
+const localStorageFetchedFeaturesKey = `__bucket_fetched_features`;
+const localStorageOverridesKey = `__bucket_overrides`;
+
+function setOverridesCache(overrides: Record<string, boolean>) {
```

Comment:
Let's pull the overrides type out into a reusable type of it's own since it's used multiple times
`export type OverridesFeatures = Record<string, boolean>`